### PR TITLE
cluster picking: implement /stock_issue

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Comment by guewen** [comment]
one thing remains to do: reassign at the end in case the product is available elsewhere

**Comment by guewen** [comment]
> one thing remains to do: reassign at the end in case the product is available elsewhere

done

File: shopfloor/actions/inventory.py [hunk 1]
```diff
@@ -20,11 +20,105 @@ def create_draft_check_empty(self, location, product, ref=None):
...
+            ("location_id", "=", location.id),
+            ("product_id", "=", move.product_id.id),
+            ("package_id", "=", package.id),
+            ("lot_id", "=", lot.id),
+            ("state", "=", "assigned"),
+        ]
+        return self.env["stock.move.line"].search(domain)
+
+    def _stock_issue_inventory_values(self, move, location, package, lot, line_qty):
+        inventory = self.actions_for("inventory")
```
Comment (simahawk) [inline shopfloor/actions/inventory.py]: isn't `self._stock_issue_product_description`?

**Review by simahawk — APPROVED** [review]
amazing work mate! Just a small remark ;)

**Comment by guewen** [comment]
The build is failed because of a flaky test that I fixed in #15.
So I merge this already and will merge the other right after.

**Pull request merged**